src/services/system/model.js

```diff
@@ -29,6 +29,13 @@ const systemSchema = new Schema(
 		alias: { type: String },
 		oaClientId: { type: String }, // just for oauth2-systems
 		oaClientSecret: { type: String }, // just for oauth2-systems
+		oauthconfig: {
+			client_id: { type: String },
```

Comment:
are all values here really optional? i owuld more less expect all props to be `required:true`

Reply:
Done.